src/styles/styles.css

```diff
@@ -0,0 +1,316 @@
+}
+
+.card:hover {
+  transform: scale(1.2);
+  transition: transform 300ms ease;
+}
+
+.card__content {
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 160px;
+  height: 134px;
+  background-image: url(../images/imac.jpeg);
+  margin-left: 19px;
+  margin-top: 32px;
+  background-size: cover;
+}
+
+.card__title {
+  margin-top: 40px;
+  margin-left: 16px;
+  height: 36px;
+  width: 166px;
+}
+
+.card__title-text {
+  font-size: 12px;
+  font-weight: 500;
+  color: #060b35;
+  line-height: 18px;
+}
+
+.card__title-text:hover {
+  color: #34568b;
+  transition: color 300ms ease;
+}
+
+.card__code {
+  width: 98px;
+  height: 14px;
+  margin-top: 4px;
+  margin-left: 16px;
+  line-height: 14px;
+}
+
+.card__code-text {
+  font-size: 10px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: 16px;
+  margin-top: 16px;
+  flex-wrap: wrap;
+  width: 166px;
+}
+
+.card__rating-stars {
+  display: flex;
+}
+
+.card__rating-star {
+  background-image: url(../images/star.svg);
+}
+
+.card__rating-star:nth-child(1) {
+  background-image: url(../images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.card__rating-star:nth-child(2) {
+  background-image: url(../images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.card__rating-star:nth-child(3) {
+  background-image: url(../images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.card__rating-star:nth-child(4) {
+  background-image: url(../images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.card__rating-star:nth-child(5) {
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.card__rating-star:first-child {
+  margin-left: 0;
+}
+
+.card__rating-text {
+  font-size: 10px;
+  line-height: 14px;
+  text-align: right;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 166px;
+  height: 16px;
+  margin-top: 24px;
+  margin-left: 16px;
+}
+
+.card__price-text {
+  font-size: 12px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.card__price-value {
+  font-size: 16px;
+  font-weight: 700;
+  margin-left: auto;
+  color: #060b35;
+}
+
+.card__buy-wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 166px;
+  height: 40px;
+  margin-top: 16px;
+  margin-left: 16px;
+  background-color: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+}
```

Comment:
The `background-color: #00acdc;` and `border: 1px solid #00acdc;` properties could be replaced with a CSS variable. This would make it easier to update the color scheme in the future.